src/main/java/com/endava/internship/collections/StudentList.java

```diff
@@ -1,140 +1,381 @@
 package com.endava.internship.collections;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
+import java.util.*;
 
 public class StudentList implements List<Student> {
+    private int capacity;
+    private int size = 0;
+    private Student[] array;
 
+    public StudentList() {
+        this.capacity = 10;
+        this.array = new Student[this.capacity];
+    }
+
+    public StudentList(int initialCapacity) {
+        if (initialCapacity > 0) {
```

Comment:
I believe, we can set 0 capacity for list, the we'll have an empty list

Reply:
Fixed!